docs/sources/http_api/auth.md

```diff
@@ -11,6 +11,27 @@ aliases = ["/docs/grafana/latest/http_api/authentication/"]
 
 Currently you can authenticate via an `API Token` or via a `Session cookie` (acquired using regular login or OAuth).
 
+## X-Grafana-Org-Id Header
+
+**X-Grafana-Org-Id**  is an optional property that specifies the organization to which the action is applied. If it is not set, the created key belongs to the current context org. Use this header in all requests except those regarding admin.
```

Comment:
```suggestion
**X-Grafana-Org-Id**  is an optional property that specifies the organization to which the action is applied. If it is not set, then the created key belongs to the current context org. Use this header in all requests except those regarding admin.
```

Reply:
Let us create a separate PR for this. This is a backport. Thanks.